netbox/ipam/filters.py

```diff
@@ -304,14 +304,15 @@ class IPAddressFilterSet(TenancyFilterSet, CustomFieldFilterSet, CreatedUpdatedF
         to_field_name='rd',
         label='VRF (RD)',
     )
-    device = django_filters.CharFilter(
-        method='filter_device',
-        field_name='name',
-        label='Device',
-    )
-    device_id = django_filters.NumberFilter(
-        method='filter_device',
-        field_name='pk',
+    device = django_filters.ModelMultipleChoiceFilter(
+        field_name='interface__device__name',
```

Comment:
This does not account for IP addresses assigned to interfaces belonging to virtual chassis members, which is why the local `filter_device()` method was added. The filter/method need to be adapted to pass multiple values.

Reply:
Thanks the review, I will fix later.

Reply:
Enabled filtering virtual chassis members also.